scripts/linux/publishReleasePackages.sh

```diff
@@ -239,7 +235,11 @@ publish_to_github()
 
         #Create Release Page
         url="https://api.github.com/repos/Azure/azure-iotedge/releases"
-        body=$(jq -n --arg version "$VERSION" --arg body "$(cat $WDIR/content.txt)" '{tag_name: $version, name: $version, target_commitish:"main", body: $body}')
+        reqBody='{tag_name: $version, name: $version, target_commitish:"main", draft: true, body: $body}'
+        if [[ $SKIP_UPLOAD == "false" ]]; then
+            reqBody='{tag_name: $version, name: $version, target_commitish:"main", body: $body}'
+        fi
```

Comment:
The first time edgelet build pipeline call this script to create a draft page, it doesn't upload any artifacts. For that run we want to create a draft page. Then later when the artifacts are ready and the version files are updated, we want to push the official release page and tag the commit. The tagging is done from the pipeline along with the file version updated.